aws/users/main.tf

```diff
@@ -0,0 +1,42 @@
+terraform {
+  required_version = ">= 0.11.2"
+
+  backend "s3" {}
+}
+
+provider "aws" {
+  assume_role {
+    role_arn = "${var.aws_assume_role_arn}"
+  }
+}
+
+data "terraform_remote_state" "account_settings" {
+  backend = "s3"
+
+  config {
+    bucket = "${var.namespace}-${var.stage}-terraform-state"
+    key    = "account-settings/terraform.tfstate"
+  }
+}
+
+
+data "terraform_remote_state" "root_iam" {
+  backend = "s3"
+
+  config {
+    bucket = "${var.namespace}-${var.stage}-terraform-state"
+    key    = "root-iam/terraform.tfstate"
+  }
+}
+
+data "aws_iam_account_alias" "default" {}
+
+output "account_alias" {
+  value = "${data.aws_iam_account_alias.default.account_alias}"
+}
+
+locals {
+  signin_url = "${data.terraform_remote_state.account_settings.signin_url}"
+  admin_group = ["${data.terraform_remote_state.root_iam.admin_group}", "${data.terraform_remote_state.root_iam.readonly_group}"]
```

Comment:
should it be just one admin group in the list?

Reply:
i struggled with naming. my thinking is that admin has admin & readonly (why shouldn't they). readonly people only have readonly. i'm overloading `admin` which i don't like. but i don't know if it's better to add more terms. feels like it's implied that an admin would get readonly. thoughts?

Reply:
why not use the `admin` and `readonly` group names from `data.terraform_remote_state.root_iam`, and then in the account modules add the user to one of the group or to both?

```hcl
groups  = ["${local.admin_group}, ${local.readonly_group}"]
```

Reply:
My bad. fixed.